### PR TITLE
Firebase auth

### LINKED ISSUE #2: Firebase authentication

### PR DESCRIPTION
**Pull request by romabulani**
- Login, Signup and Logout using firebase

### CODE REVIEW

File: src/firebase.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  messagingSenderId: process.env.REACT_APP_MESSAGING_SENDER_ID,
+  appId: process.env.REACT_APP_APP_ID,
+  measurementId: process.env.REACT_APP_MEASUREMENT_ID,
+};
+console.log(firebaseConfig.apiKey);
+const app = initializeApp(firebaseConfig);
+const auth = getAuth(app);
+const db = getFirestore(app);
+
+export { app, auth, db };
```
Comment (shrey27) [inline src/firebase.ts:19]: firebase integration is amazing

File: src/hooks/useSignupHandler.ts [hunk 2]
```diff
@@ -0,0 +1,219 @@
...
+  const navigate = useNavigate();
+
+  type AuthFormAndError = {
+    firstName: string;
+    lastName: string;
+    email: string;
+    password: string;
+    confirmPassword: string;
+  };
+
```
Comment (shrey27) [inline src/hooks/useSignupHandler.ts:20]: Its good that you have switched to typescript for this project

**Review by shrey27 — APPROVED** [review]
Code is well structured
UI is working as expected
PR is good to merge

File: src/hooks/useSignupHandler.ts (hunk 2 above)
Comment (romabulani) [reply src/hooks/useSignupHandler.ts:20]: Thankyou :)

**Review by Rohanmond — APPROVED** [review]
Folder struture looks good..Likes that you are using typescript

**Pull request merged**